libzeth/circuits/blake2s/blake2s.hpp

```diff
@@ -165,6 +165,7 @@ template<typename FieldT> class BLAKE2s_256 : public libsnark::gadget<FieldT>
 };
 
 } // namespace libzeth
+
```

Comment:
Not sure this is consistent with the other `hpp/tcc` files where we wrote the include on the line following the `}` closing the libzeh namespace

Reply:
Ah, I didn't realise it had been switched to that.
It seems slightly unnatural since in all other cases we have a line before `#include` blocks, and line between `} // namespace ...` and following code (hence it stood out).
Anyway changed for now to be consistent.